src/frontend/components/SøknadsSteg/Forside/FortsettPåSøknad.tsx

```diff
@@ -48,7 +48,7 @@ const ModalKnappeContainer = styled.div`
 `;
 
 const FortsettPåSøknad: FC = () => {
-    const { tekster, localeString } = useApp();
+    const { tekster, plainTekst } = useApp();
```

Comment:
Hva er forskjellen på `localeString` og `plainTekst`?

Reply:
Innførte plainTekst i denne PR'en: https://github.com/navikt/familie-ks-soknad/pull/52/files 

Innførte den for å kunne gjøre om Sanity blokker til plain tekst (altså ingen styling, formatering og slikt). Tenkte det var fint å ikke ha så mange funksjoner å forholde seg til når det kommer til å gjøre om sanitydata til tekst i appen vår. Derfor la jeg også inn støtte for å kunne ta inn sanity stringer i den funksjonen (ikke blokk), som basically gjør det samme som localeString. Den tar inn en record med alle språk og tekst, og returnerer teksten på det valgte språket.

Da har vi kun to ting å forholde oss til når vi skal gjøre om sanitydata. Det er 
1. `<TekstBlock />` som er react-componenten som rendrer også styling og formatering osv som blir satt i sanity. 
2. Funksjonen `plainTekst` som kun returnerer en plain tekst, enten det er en sanityblock uten styling/formatering på riktig språk, eller en string på riktig språk.